x11-drivers/xf86-video-intel/xf86-video-intel-9999.ebuild

```diff
@@ -55,7 +55,8 @@ DEPEND="${RDEPEND}
 
 src_configure() {
 	replace-flags -Os -O2
```

Comment:
I did some digging - it's from [here](https://gitweb.gentoo.org/repo/gentoo.git/commit/x11-drivers/xf86-video-intel?id=cff7b54bda514f77710a65a421cd210646d9de8c), so let's add a bug [reference](https://bugs.gentoo.org/582910) for it (ditto the other ebuild).